NS_Install2.sh

```diff
@@ -86,7 +88,7 @@ export INSECURE_USE_HTTP=true
 export HOSTNAME="127.0.0.1"
 export PORT="1337"
 
-cd /srv/nightscout-vps
+cd /srv/$GIT_REPOSITRY
```

Comment:
This is not going to work because all forked repositories are cloned somewhere else.